src/graphql-server/resolvers.test.js

```diff
@@ -32,6 +34,14 @@ describe('resolvers', () => {
         expect(mockGetHappDetails.mock.calls.map(c => c[0])).toEqual(['mockHappOne', 'mockHappTwo'])
       })
     })
+
+    describe('.holofuel', () => {
+      it('calls HoloFuelInterface.transactions.allComplete', async () => {
```

Comment:
would be good to get tests for all the endpoints we added here.

I think given the relative complexity of the code it would also be good to get good coverage of the `HoloFuelDnaInterface`. I'm not sure exactly the best way to do that. Maybe unit tests with mocking of `createZomeCall`?  LMK if you want touch base on that to think it through together.

Reply:
Totally agree, and they are underway.  I'l touch base with you about them soon.  Can we go ahead and merge this in the meanwhile?